flink-clients/src/main/java/org/apache/flink/client/cli/CliFrontendParser.java

```diff
@@ -611,6 +636,22 @@ public static void printHelpForSavepoint(Collection<CustomCommandLine> customCom
         System.out.println();
     }
 
+    public static void printHelpForCheckpoint(Collection<CustomCommandLine> customCommandLines) {
+        HelpFormatter formatter = new HelpFormatter();
+        formatter.setLeftPadding(5);
+        formatter.setWidth(80);
+
+        System.out.println(
+                "\nAction \"checkpoint\" triggers checkpoints for a running job or disposes existing ones.");
```

Comment:
> or disposes existing ones

Is that a typo from copying/pasting? Users shouldn't be allowed to dispose checkpoints 🤔

Reply:
Yeah, sorry for the mistake 😆